src/commons/components/progress-bar/progress-bar.js

```diff
@@ -0,0 +1,39 @@
+/*
+    ============= How to use "progressBar" component ======
+            
+    - progressData :  expected array for progressData
```

Comment:
@kamleshverma1 Can you provide an example here for this component so that it will be easy to figure it out - what parameters need to passed and what should be there data type.